app/assets/stylesheets/components/account.scss

```diff
@@ -32,6 +28,7 @@
 
     li {
       float: none;
+      width: 100%;
```

Comment:
The width 100% could also be achieved by adding `w-100` in each class="nav-item". because it is wrapped in a flex element.
https://getbootstrap.com/docs/4.3/utilities/sizing/

Reply:
Cool didn't know about this. I updated the nav-items to include the w-100 class